### ISSUE TITLE
Ability to force changes irrespectively of the state in registry

### ISSUE THREAD

**Issue by datafoo** [theory/sqitch#379]
The same way there is a `--log-only` option for `deploy` and `revert` commands, a `--no-log` or `--force` option could exist to deploy and revert changes by taking advantage of the sqitch plans but without managing the changes in a registry.

**Context behind the proposal**

I created a first sqitch project for a PostgreSQL server in order to create a set of 2 roles and 1 database which aims to set the base to later deploy another, more consequent, sqitch project.
The scripts make use of psql variables to avoid hard-coding the roles and database names and also to be able to create multiple sets of roles and databases. However, once the deployment is done for one set, I cannot deploy with different variables to create a different set because the sqitch registry consider that the deployment is already done:
```
$ sqitch deploy --set account_ownerrole=blue --set account_approle=blue_app --set account_db=blue mytarget
Deploying changes to mytarget
  + role/account ...... ok
  + role/account_app .. ok
  + database/account .. ok
$ sqitch deploy --set account_ownerrole=acme --set account_approle=acme_app --set account_db=acme mytarget
Nothing to deploy (up-to-date)
```
I suppose that I could use a dedicated registry for each set. However, in that case, I do not feel that I need to manage the changes in a registry and therefore I would like to be able to force the changes.

**Comment by theory** [comment]
Forcing changes in this way would pretty much defeat one of the main purposes of Sqitch: ensuring that the database is in a consistent, expected state. I think you might be better off managing such changes outside of Sqitch.
